e2e/tests/functional/plugins/conditionSet/conditionSet.e2e.spec.js

```diff
@@ -264,105 +270,85 @@ test.describe('Basic Condition Set Use', () => {
     await alphaGeneratorTreeItem.dragTo(conditionCollection);
     await betaGeneratorTreeItem.dragTo(conditionCollection);
 
-    const saveButtonLocator = page.locator('button[title="Save"]');
-    await saveButtonLocator.click();
+    await page.locator('button[title="Save"]').click();
     await page.getByRole('listitem', { name: 'Save and Finish Editing' }).click();
+
     await page.click('button[title="Change the current view"]');
 
     await expect(page.getByRole('menuitem', { name: /Lad Table/ })).toBeHidden();
     await expect(page.getByRole('menuitem', { name: /Conditions View/ })).toBeVisible();
     await expect(page.getByRole('menuitem', { name: /Plot/ })).toBeVisible();
     await expect(page.getByRole('menuitem', { name: /Telemetry Table/ })).toBeVisible();
   });
-  test('ConditionSet should output blank instead of the default value', async ({ page }) => {
-    //Navigate to baseURL
-    await page.goto('./', { waitUntil: 'domcontentloaded' });
-
-    //Click the Create button
-    await page.click('button:has-text("Create")');
-
-    // Click the object specified by 'type'
-    await page.click(`li[role='menuitem']:text("Sine Wave Generator")`);
-    await page.getByRole('spinbutton', { name: 'Loading Delay (ms)' }).fill('8000');
-    const nameInput = page.locator('form[name="mctForm"] .first input[type="text"]');
-    await nameInput.fill('Delayed Sine Wave Generator');
-
-    // Click OK button and wait for Navigate event
-    await Promise.all([
-      page.waitForLoadState(),
-      page.click('[aria-label="Save"]'),
-      // Wait for Save Banner to appear
-      page.waitForSelector('.c-message-banner__message')
-    ]);
+  test('ConditionSet should output blank instead of the default value when no telemetry available', async ({
```

Comment:
tried to make it clearer to those without the context within the context within the context of the OG issue